### PR TITLE
add favourite and submit order, handle in client

### PR DESCRIPTION
**Pull request by quanghm-0027**

### CODE REVIEW

**Comment by quanghm-0027** [comment]
ready

File: app/controllers/api/foods_controller.rb [hunk 1]
```diff
@@ -60,5 +63,55 @@ def save
         }
       end
     end
+
+    def add_favorite
+      food_id = params['food_id']
+      if !food_id.nil?
+        tmp = UserFavoriteFood.find_by :user_id => current_user.id, :food_id => food_id
```
Comment (hungct-1121) [inline app/controllers/api/foods_controller.rb]: dùng cú pháp mới đi em, user: current_user, food_id: food_id

**Comment by quanghm-0027** [comment]
ready

**Comment by quanghm-0027** [comment]
ready

**Comment by quanghm-0027** [comment]
ready

**Comment by hungct-1121** [comment]
merged

**Pull request merged**